reciprocalspaceship/io/common.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import warnings
+from contextlib import contextmanager
+from importlib.util import find_spec
+
+
+def set_ray_loglevel(level):
+    logger = logging.getLogger("ray")
+    logger.setLevel(level)
+    for handler in logger.handlers:
+        handler.setLevel(level)
+
+
+def check_for_ray():
+    has_ray = True
+    if find_spec("ray") is None:
+        has_ray = False
+
+        message = (
+            "ray (https://www.ray.io/) is not available..."
+            "Falling back to serial stream file parser."
```

Comment:
Oops this error message is now too specific. Change this to "Falling back to serial."